utils/logging.py

```
@@ -0,0 +1,28 @@
+import os
```

Pylint (3.3.7)

[error] Plugin 'pylint_django' is impossible to load, is it installed ? ('No module named 'pylint_django'')

(E0013)